src/main/java/com/google/cloud/teleport/newrelic/config/NewRelicPipelineOptions.java

```diff
@@ -23,8 +23,14 @@
     @Description("Batch count for sending multiple events to NewRelic in a single POST.")
     ValueProvider<Integer> getBatchCount();
 
+
     void setBatchCount(ValueProvider<Integer> batchCount);
 
+    void setFlushDelay(ValueProvider<Integer> flushDelay);
+
+    @Description("Number of seconds to wait for additional logs (up to batchCount) since the reception of the last log record in non-full batch, before flushing them to New Relic Logs.")
+    ValueProvider<Integer> getFlushDelay();
```

Comment:
I'm not sure about allowing Customer to change this, I'm worried about potential consequences, like giving a big number will increase in_memory messages until batch_count and if that is big as well :scary_parrot: 😨 
otherwise same, flush every second or so and reach rate limits

Reply:
We provide a sane default (2 seconds), but I think it's still valuable if the customer can change this: imagine for instance a customer that prefers not perform too many API calls and doesn't care about inducing a greater latency to the log forwarding, or a customer that needs 0-latency and doesn't care about doing a lot of API calls. I don't expect most customers to modify this (or the batch count, either), but providing a back door I think is still useful. 🤔

Reply:
Yeah trade off between advantages and risks. I'm okey, I don't have a strong opinion